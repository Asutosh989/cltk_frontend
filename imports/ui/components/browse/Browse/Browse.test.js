import React from 'react';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

// component:
import Browse from './Browse';

describe('Browse', () => {
	it('renders correctly', () => {

		const tree = renderer
			.create(
				<MuiThemeProvider>
					<Browse />
				</MuiThemeProvider>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
