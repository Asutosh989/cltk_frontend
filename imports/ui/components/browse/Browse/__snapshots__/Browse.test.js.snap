// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Browse renders correctly 1`] = `
<div
  className="page page-browse"
>
  <section
    className="page-head fullscreen image-bg bg-dark"
  >
    <div
      className="background-image-holder more-blur blur"
    >
      <img
        alt="background"
        className="background-image"
        src="/images/books4.jpg"
      />
    </div>
    <div
      className="background-screen light"
    />
    <div
      className="container v-align-transform"
    >
      <div
        className="row"
      >
        <div
          className="col-sm-10 col-sm-offset-1 text-center"
        >
          <h1
            className="mb40 mb-xs-16 large"
          >
            Browse
          </h1>
        </div>
      </div>
    </div>
  </section>
  <section
    className="page-content"
  >
    <div
      className="works-list"
    >
      <div
        className="reading-loading"
      >
        <div
          className="well-spinner-double"
        >
          <div
            className="double-bounce1"
          />
          <div
            className="double-bounce2"
          />
        </div>
      </div>
    </div>
  </section>
</div>
`;
